### DISCUSSION TITLE
Polygon edge - Chainbridge

### DISCUSSION THREAD

**Discussion by reddigszymon** [0xPolygon/polygon-edge#934] [General]
# Trying to create a bridge between my Polygon Edge Blockchain and Polygon Mumbai 
## Description
I am trying to create a bridge between mumbai and my polygon edge network, but when I try to turn on the relayer it "cannot fetch the code" from my blockchain (even though I deployed all the necessary contracts).

## Your environment
- OS and version: MacOS 13.0.1 (22A400)
- The version of the Polygon Edge.    
  (*Confirm the version of your Polygon edge client by running the following command: `polygon-edge version --grpc-address GRPC_ADDRESS`*) when i run the polygon-edge version i get <none> in every parameter (even though it works, lol)
- The branch that causes this issue.
- Locally or Cloud hosted (which provider). locally
- Please confirm if the validators are running under containerized environment (K8s, Docker, etc.). all the validators are running correctly

## Steps to reproduce
- Tell us how to reproduce this issue.
- Where the issue is, if you know.
- Which commands triggered the issue, if any.
- Provide us with the content of your genesis file.
- Provide us with commands that you used to start your validators.
- Provide us with the peer list of each of your validators by running the following command: `polygon-edge peers list --grpc-address GRPC_ADDRESS`.
- Is the chain producing blocks and serving customers atm?

The chain is normally producing blocks, i went through the whole tutorial in the docs about chainbridge, but I am stuck at the end part of turning on the relayer. It says it CANNOT FETCH CODE on my local blockchain.

## Expected behavior
- Tell us what should happen.
- Tell us what happened instead.

## Logs
Provide us with debug logs from all of your validators by setting logging to `debug` output with: `server --log-level debug`

## Proposed solution
If you have an idea on how to fix this issue, please write it down here, so we can begin discussing it

Here is the picture

![image](https://user-images.githubusercontent.com/63747414/202713297-262ba799-152d-400d-a6b3-6e2e56a5fdb7.png)

**Comment by ivanbozic21** [comment]
Hi @reddigszymon 

Please know that this is Polygon Edge issue board, and for that error message you will have to submit an issue to Chainbridge repository board if you still receive this error message correctly following our Chainbridge instructions:

https://github.com/ChainSafe/chainbridge/

**Comment by reddigszymon** [comment]
Thanks for the reply @ivanbozic21 although chainbridge's repo is archived and there is no way to file an issue.
